content.js

```diff
@@ -163,11 +163,14 @@ if (window.location.href.startsWith("https://www.purdue.edu/apps/account/cas/log
     }
 }
 
+//Skip Brightspace campus selection screen
```

Comment:
will this break the extension for non-WL users?

Reply:
The "Purdue West Lafayette" and "Purdue Northwest" buttons both go to the same url, so users from those campuses should be fine. I don't think the other two campuses use Boilerkey so there isn't really a reason for them to have the extension.

It behaves the same way it did before a website/chrome update broke the original redirect code.

Reply:
The original redirect function was added in PR #11

Reply:
Sounds good to me, I'll publish tomorrow!